conseil-api/src/main/resources/logging.conf

```diff
@@ -1,5 +1,6 @@
 logging: {
     muted: false
+    output-json-with-service-name: ${?CONSEIL_LOG_JSON_SERVICE_NAME}
```

Comment:
can you put here the default value of this variable? from this I can't tell if it's boolean or something else

Reply:
Hmm, right now setting this to any value will trigger json logging which is not great

I'll add another field `enable-json-output` that will work like a on/off switch, that will make it clearer